internal/provider/attribute_plan_modifier/default_value.go

```diff
@@ -91,7 +89,7 @@ func (apm *readyForRenewalAttributePlanModifier) Modify(ctx context.Context, req
 		return
 	}
 
-	if earlyRenewalHours.Value >= validityPeriodHours.Value {
+	if earlyRenewalHours.ValueInt64() >= validityPeriodHours.ValueInt64() {
 		res.AttributePlan = types.Bool{
 			Value: true,
 		}
```

Comment:
I think the linters have a harder time finding inline field declarations of these -- I'd suggest searching for `Value:`, `Null:`, and `Unknown:` to tease these out as well. 👍 

```suggestion
		res.AttributePlan = types.BoolValue(true)
```

Reply:
Have done a global search to catch deprecations not caught by `golangci-lint`.